Vimeo-DotNet/VimeoAPI/VimeoAPI.csproj

```diff
@@ -32,6 +32,7 @@
     <WarningLevel>4</WarningLevel>
     <Prefer32Bit>false</Prefer32Bit>
     <LangVersion>latest</LangVersion>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Comment:
Did we want this when your configuration is `Release` as well:
![image](https://user-images.githubusercontent.com/3475163/43732707-605687cc-9980-11e8-9c3a-f65b20efd473.png)

:smiley:

Reply:
Yes, changed it